BlockchainSdk/BlockchainService/Aptos/AptosNetworkService.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  AptosNetworkService.swift
+//  BlockchainSdk
+//
+//  Created by skibinalexander on 29.01.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import Combine
+import SwiftyJSON
+
+class AptosNetworkService: MultiNetworkProvider {
+    // MARK: - Protperties
+    
+    let blockchain: Blockchain
```

Comment:
Давай лучше без любого)
Все равно передача другого blockchain не поможет)
```suggestion
    let blockchain: Blockchain = .autos
```

Reply:
у нас что-то это постоянно всплывает, с какого файла копипаста? Может надо там исправить

Reply:
Да тащу ошибку , буду обращать внимание 👌